src/lib/WalletInfoCollector.ts

```diff
@@ -128,7 +129,7 @@ export default class WalletInfoCollector {
             WalletInfoCollector._keyguardClient = WalletInfoCollector._keyguardClient || new KeyguardClient();
         } else if (walletType === WalletType.LEDGER) {
             WalletInfoCollector._wasmInitializationPromise =
-                WalletInfoCollector._wasmInitializationPromise || Nimiq.WasmHelper.doImportBrowser();
+                WalletInfoCollector._wasmInitializationPromise || Nimiq.WasmHelper.doImport();
```

Comment:
@danimoh I believe with your changes in core in 1.4.3 this is now simpler to track, as `doImport()` always returns the same promise, right? So `_wasmInitializationPromise` is not needed anymore.